src/puppet-dom.js

```diff
@@ -50,7 +35,7 @@
       this.element.removeEventListener('puppet-redirect-pushstate', this.historyHandler);
     };
 
-    options.remoteUrl = options.remoteUrl || getRemoteUrlFromCookie() || window.location.href;
+    options.remoteUrl = options.remoteUrl || window.location.href;
```

Comment:
Fair point, but please not that we use `location` in many other places, and I would say it's quite useful to have notion of location even in node version